src/main/java/org/broadinstitute/dsm/route/CreateBSPDummyKitRoute.java

```diff
@@ -0,0 +1,60 @@
+package org.broadinstitute.dsm.route;
+
+import org.apache.commons.lang3.StringUtils;
+import org.broadinstitute.ddp.handlers.util.Result;
+import org.broadinstitute.dsm.db.DDPInstance;
+import org.broadinstitute.dsm.db.KitRequestShipping;
+import org.broadinstitute.dsm.db.dao.kit.BSPKitDao;
+import org.broadinstitute.dsm.statics.RequestParameter;
+import org.broadinstitute.dsm.util.DBUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import spark.Request;
+import spark.Response;
+import spark.Route;
+
+public class CreateBSPDummyKitRoute implements Route {
+    private static final String DUMMY_KIT_TYPE_NAME = "DUMMY_KIT_TYPE";
+    private static final String DUMMY_REALM_NAME = "DUMMY_KIT_REALM";
+    private static final String USER_ID = "74";
+    private static final Logger logger = LoggerFactory.getLogger(CreateBSPDummyKitRoute.class);
+
+    @Override
+    public Object handle(Request request, Response response) throws Exception {
+        logger.info("Got Mercury Test request");
+        String kitLabel = request.params(RequestParameter.LABEL);
+        if (StringUtils.isBlank(kitLabel)) {
+            response.status(400);// return bad request
+            logger.error("Bad request from Mercury! Should include a kitlabel");
+            return new Result(400, "Please include a kit label as a path parameter");
```

Comment:
let's step by step move from using `Result` to using `Response` itself, if you want to return message you can simple return `String`.